### PR TITLE
Proper endpoints in integration tests + 2 negative test cases

### PR DESCRIPTION
**Pull request by tisnik**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] Integration tests

## Description

- Proper endpoints to be checked in integration tests
- Added two negative test cases

## Related Tickets & Documents

- Related Issue #[OLS-83](https://issues.redhat.com//browse/OLS-83)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Unit tests passed locally.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: tests/integration/test_ols.py [hunk 1]
```diff
@@ -46,6 +46,23 @@ def test_feedback() -> None:
...
+    assert response.status_code == requests.codes.unprocessable_entity
+
+
+def test_feedback_wrong_not_filled_in_request() -> None:
+    # for the request send w/o proper payload, the server
+    response = client.post(
+        "/feedback", json={"conversation_id": 0, "feedback_object": None}
+    )
+    # for the request send w/o proper payload, the server
+    # should respond with proper error code
```
Comment (onmete) [inline tests/integration/test_ols.py:62]: comments are a bit mixed

Comment (tisnik) [reply tests/integration/test_ols.py:61]: nice catch! fixed

File: tests/integration/test_ols.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def test_healthz() -> None:
 
 
 def test_readyz() -> None:
-    response = client.get("/healthz")
```
Comment (thoraxe) [inline tests/integration/test_ols.py:17]: But we support both in app.py - shouldn't we test both? This just changes which one we test.

File: tests/integration/test_ols.py [hunk 3]
```diff
@@ -29,7 +29,7 @@ def test_root() -> None:
 
 
 def test_status() -> None:
-    response = client.get("/")
+    response = client.get("/status")
```
Comment (thoraxe) [inline tests/integration/test_ols.py:32]: See previous comment about healthz

File: tests/integration/test_ols.py [hunk 4]
```diff
@@ -46,6 +46,22 @@ def test_feedback() -> None:
     assert response.json() == {"status": "feedback received"}
 
 
+def test_feedback_wrong_request() -> None:
```
Comment (thoraxe) [inline tests/integration/test_ols.py:49]: Feedback has its own endpoint. Shouldn't this be in a separate test file?

File: tests/integration/test_ols.py [hunk 5]
```diff
@@ -46,6 +46,22 @@ def test_feedback() -> None:
...
+    response = client.post("/feedback", json={})
+    # for the request send w/o proper payload, the server
+    # should respond with proper error code
+    assert response.status_code == requests.codes.unprocessable_entity
+
+
+def test_feedback_wrong_not_filled_in_request() -> None:
```
Comment (thoraxe) [inline tests/integration/test_ols.py:56]: See previous Re separate file

File: tests/integration/test_ols.py (hunk 2 above)
Comment (tisnik) [reply tests/integration/test_ols.py:17]: we do test both. /healtz is tested in `test_healthz` just few lines above. Basically these two functions call the one API

File: tests/integration/test_ols.py (hunk 3 above)
Comment (tisnik) [reply tests/integration/test_ols.py:32]: dtto, there's function `test_root` for it

File: tests/integration/test_ols.py (hunk 4 above)
Comment (tisnik) [reply tests/integration/test_ols.py:49]: if yes, then I'll move the `test_feedback` there to. Is it ok/expected?

File: tests/integration/test_ols.py (hunk 5 above)
Comment (tisnik) [reply tests/integration/test_ols.py]: @thoraxe now it is separated

**Review by thoraxe — APPROVED** [review]

**Comment by thoraxe** [comment]
Please force push a squashed commit

**Pull request merged**